### ISSUE TITLE
Documentation missing

### ISSUE THREAD

**Issue by DerAndere1** [gbl/D7CReader#1]
I could not find out how to use this program exactly. Please give a short description of features and usage. From the comments it looks like a command line interface that accepts an input file path as argument. I guess it has to be compiled with JDK and then can be run like this from the command line (cmd.exe on Windows)?: 
`d7creader "path\to\inputfile\filename.stp"`

can it read .stp files for knitting machines or .pat files for knitting machines or both?
Thanks

**Comment by gbl** [comment]
Well, the program isn't really intended to be used as it is; rather, I wrote it to help someone who asked how to read .stp files, presumably into some other kind of software than the original one. It's more intended to be used as an example for someone who wants to write their own program than to be used to actually do anything with stp files.

I added a README which includes information about how to compile and run the program. Yes, you're right, you run it from the command line, pass the name of your filename.stp as parameter, and get an ascii dump of the file contents. This probably won't help you much, though :/
